pgmpy/factors/CPD.py

```diff
@@ -117,6 +117,21 @@ def __init__(self, variable, variable_card, values,
             raise TypeError("Values must be a 2D list/array")
 
         super(TabularCPD, self).__init__(variables, cardinality, values.flatten('C'))
+        
+    def __repr__(self):
+        if not self.evidence:
+            str_1 = ""
+            str_1 += '<TabularCPD representing P(' + str(self.variable) + ':' + str(self.variable_card) + ')>'
```

Comment:
Why are you creating an empty string ```str_1``` here if you are going to add something to it?